src/main/java/hudson/plugins/git/GitChangeSet.java

```diff
@@ -444,6 +485,22 @@ private boolean isCreateAccountBasedOnEmail() {
         return descriptor.isCreateAccountBasedOnEmail();
     }
 
+    private boolean isUseExistingAccountWithSameEmail() {
+        DescriptorImpl descriptor = getGitSCMDescriptor();
+
+        if (descriptor == null) {
+            return false;
+        }
+
+        return descriptor.isUseExistingAccountWithSameEmail();
+    }
+
+    @SuppressFBWarnings(value="NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE",
+        justification="Jenkins.getInstance() is not null")
+    private DescriptorImpl getGitSCMDescriptor() {
+        return (DescriptorImpl) Hudson.getInstance().getDescriptor(GitSCM.class);
```

Comment:
Since the findbugs exclusion mentions Jenkins, could the reference mention Jenkins.getInstance() instead of Hudson.getInstance()?

Reply:
corrected